pkg/valueset/postgresql/postgresql_driver.go

```diff
@@ -0,0 +1,66 @@
+package postgresql
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jackc/pgx/v5"
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+// match (n:OMOPConcept) return n
+func GetResults(db Database, queryParams map[string]string) (map[string]string, error) {
```

Comment:
Pass context to this
GetResults should get a tableId arg, and just look at that table